src/config/inversion-fixes.config

```diff
@@ -349,6 +349,7 @@ chat.openai.com
 
 INVERT
 .overflow-hidden:has(textarea)
+pre .dark
```

Comment:
Why to INVERT something that is good.
Check how NO INVERT section looks like in this file and apply that here.

I could do that by myself but if you will make that change you will learn something new. 😉